src/backend/src/lib.rs

```diff
@@ -265,6 +266,12 @@ async fn caller_btc_address(network: BitcoinNetwork) -> String {
     public_key_to_p2pkh_address(network, &ecdsa_pubkey_of(&ic_cdk::caller()).await)
 }
 
+/// Returns the balance of the given Bitcoin address.
+#[query]
+async fn btc_balance(address: String, network: BitcoinNetwork) -> u64 {
+    bitcoin_lib::get_balance(network, address).await
```

Comment:
nth: just `get_balance` and declare the `use` at the top for consistency?

Reply:
yes, agreed!